src/v/redpanda/admin/api-doc/debug_bundle.json

```diff
@@ -0,0 +1,31 @@
+"/v1/debug/bundle": {
+  "post": {
+    "description": "Posting to this endpoint will trigger the start of an rpk debug bundle process.  If a process is already executing, this endpoint will return a 409 error.  If there is an issue with starting the process, this endpoint will return a 500 error.\n\nNote if job_id is not provided in BundleStartConfig, one will be automatically generated and returned in BundleStartResponse.",
```

Comment:
The last part (about the Job ID) is incorrect (that's my fault).  The JSON schema indicates that job id is required but the description was wrong.  My bad.  I just updated the API page

Reply:
Done

Reply:
Maybe you updated this later but the description is the old one

Reply:
Ah it got changed in a later commit